src/colejorz/pilothouse.py

```diff
@@ -36,7 +39,7 @@ class Pilothouse:  # pylint:disable=too-many-instance-attributes
     STOP = 'stop'
     BACKWARD = 'backward'
 
-    def __init__(self, queue):
+    def __init__(self, queue: "Queue[Union[int, Dict[str, int]]]") -> None:
```

Comment:
`TypeError: 'type' object is not subscriptable` occurs during runtime. I followed this comment: https://github.com/ClearcodeHQ/colejorz/pull/23#discussion_r319401868

Reply:
hmmm... typevar Queue instead?

Reply:
According to https://mypy.readthedocs.io/en/latest/common_issues.html there are two solutions:

> Some classes are declared as generic in stubs, but not at runtime. Examples in the standard library include os.PathLike and queue.Queue. Subscripting such a class will result in a runtime error:

```
from queue import Queue

class Tasks(Queue[str]):  # TypeError: 'type' object is not subscriptable
    ...

results: Queue[int] = Queue()  # TypeError: 'type' object is not subscriptable
```

> To avoid these errors while still having precise types you can either use string literal types or typing.TYPE_CHECKING:


```
from queue import Queue
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    BaseQueue = Queue[str]  # this is only processed by mypy
else:
    BaseQueue = Queue  # this is not seen by mypy but will be executed at runtime.

class Tasks(BaseQueue):  # OK
    ...

results: 'Queue[int]' = Queue()  # OK
```